pkg/controller/atlasconnection/atlasconnection_controller.go

```diff
@@ -73,12 +73,12 @@ type MongoDBAtlasConnectionReconciler struct {
 
 // Dev note: duplicate the permissions in both sections below to generate both Role and ClusterRoles
 
-// +kubebuilder:rbac:groups=atlas.mongodb.com,resources=MongoDBAtlasConnections,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=atlas.mongodb.com,resources=MongoDBAtlasConnections/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=dbaas.redhat.com,resources=mongodbtlasconnections,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=dbaas.redhat.com,resources=mongodbtlasconnections/status,verbs=get;update;patch
```

Comment:
isn't the resource here missing an 'a'? mongodb_a_tlasconnections?

Reply:
Good catch. Will be corrected